src/context.rs

```diff
@@ -127,22 +121,14 @@ impl<A: Actor> Context<A> {
         ctx.run(actor)
     }
 
-    /// Stop the actor as soon as it has finished processing current message. This will mean that the
-    /// [`Actor::stopping`](trait.Actor.html#method.stopping) method will be called.
-    pub fn stop(&mut self) {
-        self.running = RunningState::Stopping;
+    /// Stop this actor as soon as it has finished processing current message. This means that the
+    /// [`Actor::stopped`] method will be called.
+    pub fn stop_self(&mut self) {
+        self.running = false;
     }
```

Comment:
This is a breaking change that I am not sure is necessary?

There is no semantic change here, is there? People relying on `stopping` will already notice that things have changed.

Reply:
It's true that those relying on `stopping` will already see the change, but since `stop_all` is now becoming public, I thought it might be better to rename this and clarify that it stops only the current actor as opposed to all on the address.

Reply:
I didn't see `stop_all` becoming public, makes total sense then!